README.md

```diff
@@ -4,7 +4,25 @@
 
 A Rust wrapper library for devkitPro's [libogc](https://github.com/devkitPro/libogc).
 
-See the [Wii testing project](https://github.com/rust-wii/testing-project) for instructions on how to use this library.
+To get started, you'll first need to install the following dependencies on your system:
+* [Rust, along with rustup and cargo](https://www.rust-lang.org/tools/install)
+* Clang compiler
```

Comment:
Where did you install Clang from? Is it safe to assume you're on Linux and just grabbed it with your package manager?

Reply:
yep, i'm on ubuntu and for me it was just `sudo apt install clang`.

i'll update it to be a bit more descriptive and include a link to LLVM's page for getting clang just in case